### DISCUSSION TITLE
Python Surrogates

### DISCUSSION THREAD

**Discussion by jucavel09** [snl-dakota/dakota#96] [Q&A]
I am trying to run one of the python examples in \share\dakota\examples\official\surrogates\library\test_notebook.py

I'm trying to import the surrogates module using:
import dakota.surrogates as daksurr
When I look in the folder \share\dakota\Python\dakota it doesn't seem like this exists.

In the installation instructions it says if you add that to your PYTHONPATH env variables it should be able to import the surrogate models.

Is there a way to generate and run surrogate models through python?

**Comment by jadamstephens** [comment]
We enable that feature in our binary downloads only for RHEL7 and RHEL8. Also, it's not on by default for a source build. Some help for building from source is [here](https://snl-dakota.github.io/docs/6.19.0/users/compiling/compiling.html).
